pkg/kv/store.go

```diff
@@ -146,14 +148,14 @@ func UnregisterAllDrivers() {
 
 // Open lookup driver with 'name' and return Store based on 'dsn' (data source name).
 // Failed with ErrUnknownDriver in case 'name' is not registered
-func Open(ctx context.Context, name, dsn string) (Store, error) {
+func Open(ctx context.Context, name string, params kvparams.KV) (Store, error) {
 	driversMu.RLock()
 	d, ok := drivers[name]
 	driversMu.RUnlock()
```

Comment:
I know it's not part of your PR, but it caught my eye
```suggestion
	defer driversMu.RUnlock()
```